console/backend/src/main/java/org/frankframework/management/web/Init.java

```diff
@@ -41,25 +40,27 @@
 @RestController
 public class Init extends FrankApiBase {
 
-	@Autowired
-	protected @Getter HttpServletRequest servletRequest;
+	private final RequestMappingHandlerMapping handlerMapping;
 
-	@Autowired
-	private RequestMappingHandlerMapping handlerMapping;
+	public Init(RequestMappingHandlerMapping handlerMapping) {
+		this.handlerMapping = handlerMapping;
+	}
 
 	private boolean isMonitoringEnabled() {
 		return getProperty("monitoring.enabled", false);
 	}
 
 	@GetMapping(value = {"", "/"}, produces = "application/json")
 	@PermitAll
-	public ResponseEntity<?> getAllResources(@RequestParam(value = "allowedRoles", required = false) boolean displayAllowedRoles, @RequestParam(value = "hateoas", defaultValue = "default") String hateoasImpl) {
+	public ResponseEntity<?> getAllResources(HttpServletRequest servletRequest,
```

Comment:
HttpServletRequest kan je op deze manier door spring in een contrller method laten injecteren.